src/Controllers/ErrorController/index.js

```diff
@@ -0,0 +1,5 @@
+function ErrorController() {
+    throw 'Error: Stop doing that'
```

Comment:
haha as great as this is, it would be best to just respond with JSON in the same format as the other endpoints. Just set success to false, and out in a message like 'this HTTP method is not supported on this URL'